test/syscall/syscall.py

```diff
@@ -628,7 +628,12 @@ def run(self):
         errfile = open(self.errfilename, 'w')
         if self.args.verbose:
             print('RUNNING: ' + str(callargs))
-        subret = subprocess.call(callargs, stdout=outfile, stderr=errfile)
+        # We have to overload the WIREDTIGER_CONFIG to ensure we perform
+        # fdatasync operations on checkpoints
+        my_env = os.environ.copy()
+        my_env["WIREDTIGER_CONFIG"] = "checkpoint_sync=true"
```

Comment:
I've made the change Michael suggested. The ```WIREDTIGER_CONFIG``` config mentioned is set Jenkins wide. We could remove it, but that would make it removed from all Jenkins jobs.  At the moment, I lean towards having this over-ride here, as having ```checkpoint_sync``` set either on or off is an implicit assumption for the task.